app/web/features/search/SearchResultsList.tsx

```diff
@@ -48,7 +48,7 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: theme.palette.background.paper,
     boxShadow: "0 0 4px rgba(0,0,0,0.25)",
     marginBottom: theme.spacing(3),
-    scrollMarginTop: "200px",
+    scrollMarginTop: theme.shape.navPaddingSmUp,
```

Comment:
What happens if the screen size is small?

Take a closer look at other places this variable is being used and how it's handled on different screen sizes.

We usually have one case for Xs and one for SmUp.


Also just FYI, the map is currently being re-written, so this code will be obsolete soon. Totally fine if you want to work on it as a learning exercise, but if you want to pick up something else no worries there. Or if you need help picking something just ping me!

Reply:
@joachimageron Do you wanna go ahead and close this PR then? Or do you plan to add more to it?